lib/workers/repository/init/merge.ts

```diff
@@ -27,16 +27,21 @@ export async function detectRepoFileConfig(): Promise<RepoFileConfig> {
   const cache = getCache();
   let { configFileName } = cache;
   if (configFileName) {
-    let configFileParsed = (await platform.getJsonFile(configFileName))!;
+    let configFileRaw: string | undefined = (await platform.getRawFile(
```

Comment:
is the `platform.getJsonFile` now unused? should we remove it after this PR?

Reply:
actually it is used in 3 other places.

https://github.com/renovatebot/renovate/blob/e5c6f38d750383538bab95a40184058fec5bf0e0/lib/workers/repository/onboarding/branch/check.ts#L68-L73

https://github.com/renovatebot/renovate/blob/8270d5512d86bca8eb3961ed01a3034cb24d243d/lib/workers/repository/process/index.ts#L40-L44

and part if `lib/workers/repository/init/apis.ts` as part of an helper function that being used in some other un-exported functions

https://github.com/renovatebot/renovate/blob/e5c6f38d750383538bab95a40184058fec5bf0e0/lib/workers/repository/init/apis.ts#L20-L26